engine/lib/membrane_rtc_engine/events/track_variant_paused.ex

```diff
@@ -10,8 +10,11 @@ defmodule Membrane.RTC.Engine.Event.TrackVariantPaused do
   @typedoc """
   Type describing TrackVariantPaused event.
   """
-  @type t :: %__MODULE__{variant: Track.variant()}
+  @type t :: %__MODULE__{
+          variant: Track.variant(),
+          reason: :inactive | :muted
```

Comment:
Can we describe the difference between those two?

Reply:
The track variant can be paused due to two reasons.
`:inactive` means that `VariantTracker` has detected reduced or no activity of the variant and has set it to this state
`:muted` means that we have explicitly paused variant, indicating that the lack of data sent is deliberate and that we may resume it in the future

Reply:
Yeah but let's write this in docs